plyer/platforms/ios/gyroscope.py

```diff
@@ -12,7 +12,7 @@ class IosGyroscope(Gyroscope):
     def __init__(self):
         super(IosGyroscope, self).__init__()
         self.bridge = autoclass('bridge').alloc().init()
-        self.bridge.motionManager.setGyroscopeUpdateInterval_(0.1)
```

Comment:
I think it is better to figure out the version of api in which this was changed and then add a if condition...